backend/src/libs/packages/open-ai/open-ai.package.ts

```diff
@@ -37,6 +38,21 @@ class OpenAI {
     return this.parseCompletionResponse(text);
   }
 
+  public async createImage(prompt: string): Promise<string> {
+    const { data } = await this.openAIApi.createImage({
+      prompt,
+      response_format: 'b64_json',
+    });
+
+    const [image] = data.data;
+
+    const buffer = Buffer.from(image?.b64_json ?? '', 'base64');
```

Comment:
Maybe should move it to file package

Reply:
agreed

Reply:
i think we will use it only in this place. at lease for now. let's